python_cross_site_scripting/HelloWorldWinning_vps/u8.py

```
@@ -0,0 +1,96 @@
+            continue
+
+        relative_path = '/'.join([secure_filename(part) for part in file.filename.split('/')])
+        save_path = os.path.join(base_upload_path, relative_path)
+
+        os.makedirs(os.path.dirname(save_path), exist_ok=True)
+        file.save(save_path)
+
+    return f'Files uploaded successfully to {base_upload_path}'
+
+if __name__ == '__main__':
+    app.run(debug=True, host='0.0.0.0', port=777)
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)